locales/fa/json.json

```diff
@@ -368,7 +368,7 @@
     "Lithuania": "لیتوانی",
     "Load :perPage More": "بارگذاری :perPage دیگر",
     "Loading": "در حال بارگذاری",
-    "Location": "Location",
+    "Location": "محل",
```

Comment:
```suggestion
    "Location": "موقعیت",
```

Reply:
Can you embrace these changes? Do you have that option available to you?

![image](https://github.com/Laravel-Lang/lang/assets/10347617/0f51b9ba-5e2c-4920-a172-10c3e67f26b1)